services/src/floodlight-modules/src/test/java/org/openkilda/floodlight/config/Config.java

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight.config;
+
+import org.openkilda.floodlight.switchmanager.SwitchManagerConfig;
+
+public class Config implements SwitchManagerConfig {
```

Comment:
Please, add Test prefix/suffix to distinguish from production code.

Reply:
sure